vote/cnab/app/charts/vote/templates/redis-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redis
+  namespace: {{ .Release.Namespace }}
+  labels:
+    # The "heritage" label is used to track which tool deployed a given chart.
+    # It is useful for admins who want to see what releases a particular tool
+    # is responsible for.
+    heritage: {{ .Release.Service }}
+    # The "release" convention makes it easy to tie a release to all of the
+    # Kubernetes resources that were created as part of that release.
+    release: {{ .Release.Name }}
+    # This makes it easy to audit chart usage.
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app: {{ template "vote.name" . }}
+
+spec:
+  replicas: 1
+  template:
```

Comment:
The chart install failed for me on account of the `app` label missing from `spec.template.metadata.labels`.  Once I added it, the chart installed successfully.
```
Error: release voteapp-vote failed: Deployment.apps "redis" is invalid: [spec.selector: Required value, spec.template.metadata.labels: Invalid value: map[string]string(nil): `selector` does not match template `labels`]
```
Added:
```
spec:
  replicas: 1
  template:
    metadata:
      labels:
        app: {{ template "vote.name" . }}
...
```